src/Employee/Employee.java

```diff
@@ -0,0 +1,43 @@
+package Employee;
+
+public class Employee {
+    private double salary = 759.35;
+    public String surname = "Johnson";
+    int id = 56;
+
+    public void clientInfo() {
+        System.out.println(surname);
+    }
+
+    public void info() {
+        System.out.println("Bank Accaunt id:" + id + "\nSalary:" + salary);
+    }
+
+    public void cash() {
+        System.out.println(salary);
+    }
+
+    public Employee(String surname) {
+        this.surname = surname;
+    }
+
+    Employee(int ID) {
```

Comment:
Please follow java code convention for variables naming - id instead of ID. Upper case is used for constants declaration